js/filters.js

```diff
@@ -44,23 +42,36 @@ const setfeaturesFilter = (filterElement, mapEntries) => {
   const filter = filterElement.value;
   mapEntries.forEach((entry) => {
     entry.filters[filter] = filterElement.checked
-      ? !entry.data.offer.features.includes(filter)
+      ? !entry.ad.offer.features.includes(filter)
       : false;
   });
 };
 
-const runFilter = (map, mapEntries) => {
+const runFilters = (map, mapEntries) => {
   mapEntries.forEach((entry) => {
     const filters = Object.values(entry.filters);
     if (filters.includes(true)) {
       removeFromMap(map, entry.marker);
     } else {
-      putToMap(entry.data, map, entry.marker);
+      putToMap(entry.ad, map, entry.marker);
     }
   });
 };
 
-export const setFilters = (filtersFormElements, map, mapEntries) => {
+export const addFiltration = () => ({
```

Comment:
Название вводит в заблуждение. функция разве добавляет? - Такая функция по смыслу  - конструктор
```suggestion
export const createFiltration = () => ({
```